var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -115,6 +117,8 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
     generator("ninja")
     depends_on("ninja@1.10:", type="build")
 
+    patch("fjfrt_mod.patch", when="%fj")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
@alalazo 
Added a comment.